clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -870,7 +870,37 @@ synchronized public AlterReplicaLogDirsResult alterReplicaLogDirs(
     @Override
     synchronized public DescribeLogDirsResult describeLogDirs(Collection<Integer> brokers,
```

Comment:
This implementation is provided because I wanted to use MockAdminClient rather than use Mockito and mock the Admin interface. I am open to suggestions for improving the current logic.

Reply:
That's fine. I would suggest to look at JmxToolTest for a way to get out/err content and assert on it, which is what the original test does.